tests/unit/ConversionSummary.spec.ts

```diff
@@ -0,0 +1,96 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import sinon from 'sinon'
+import { getVuexStore } from '@/store/'
+import { createLocalVue, mount } from '@vue/test-utils'
+import ConversionSummary from '@/components/Conversion/ConversionSummary.vue'
+import NatureOfBusiness from '@/components/common/YourCompany/NatureOfBusiness.vue'
+import OfficeAddresses from '@/components/common/YourCompany/OfficeAddresses.vue'
+import { GeneralPartnershipResource } from '@/resources/Change/GeneralPartnershipResource'
+
+Vue.use(Vuetify)
+
+const localVue = createLocalVue()
+const vuetify = new Vuetify({})
+
+describe('Conversion Summary component', () => {
```

Comment:
This is the same as ChangeSummary.spec.ts for now, but will need to be updated when ConversionSummary component is developed in the near future.